src/timepicker/timepicker.js

```diff
@@ -26,6 +27,11 @@ angular.module('ui.bootstrap.timepicker', [])
       this.setupMousewheelEvents( hoursInputEl, minutesInputEl );
     }
 
+    var arrowkeys = angular.isDefined($attrs.arrowkeys) ? $scope.$eval($attrs.arrowkeys) : timepickerConfig.arrowkeys;
```

Comment:
I meant modify the directive scope to set `arrowkeys: '=?'`.  It should be present by the time the init function is run, so you can directly pull the value from `$scope.arrowkeys` in that case.